src/Tribe/Editor/Attendee_Registration.php

```diff
@@ -34,7 +34,17 @@ public function hook() {
 	 * @return string
 	 */
 	public function filter_admin_body_class( $classes ) {
-		$ar_page_slug = tribe( 'tickets.attendee_registration' )->get_slug();
+		/**
+		 * This Try/Catch is present to deal with a problem on Autoloading from version 5.1.0 ET+ with ET 5.0.3.
+		 *
+		 * @todo Needs to be revised once proper autoloading rules are done for Common, ET and ET+.
+		 */
+		try {
+			$ar_page_slug = tribe( 'tickets.attendee_registration' )->get_slug();
+		} catch( RuntimeException $error ) {
```

Comment:
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after CATCH keyword; 0 found

<sub>hash: 27d4f15c70fe111351269df589b2d7e9</sub>